tests/generative/variable_strategies.py

```diff
@@ -164,6 +165,9 @@ def exists(_type, _frame):
     def count(_type, _frame):
         return st.builds(AggregateByPatient.Count, any_frame())
 
+    def count_distinct(_type, _frame):
+        return aggregation_operation(any_type(), AggregateByPatient.CountDistinct)
```

Comment:
Ughh, so my "correction" was itself incorrect. This Hypothesis code can be really hard to reason about because it's easy to forget whether you're working with a value of a certain type, or a _strategy_ for generating values of that type.

In any case, I've actually tested the below this time and I'm pretty sure it's what we want.
```suggestion
    @st.composite
    def count_distinct(draw, _type, _frame):
        type_ = draw(any_type())
        frame = draw(many_rows_per_patient_frame())
        return AggregateByPatient.CountDistinct(draw(series(type_, frame)))
```